requirements/dev.txt

```diff
@@ -21,7 +21,7 @@ bleach==5.0.1
     # via
     #   -r requirements/test.txt
     #   readme-renderer
-build==0.8.0
+build==0.9.0
```

Comment:
Dropped support for `Python 3.6` in the new version https://pypa-build.readthedocs.io/en/stable/changelog.html#id1